datarepo-clienttests/src/main/resources/servers/shdev.json

```diff
@@ -16,6 +16,6 @@
   },
   "testRunnerServiceAccountFile": "jade-k8-sa.json",
   "skipDeployment": true,
-  "skipKubernetes": false,
```

Comment:
This is just for my personal dev deploy setup - I've been meaning to change this, so I'm going to leave this in.